cmake/Modules/FindMKL.cmake

```diff
@@ -19,7 +19,13 @@ caffe_option(MKL_USE_STATIC_LIBS "Use static libraries" OFF IF NOT MKL_USE_SINGL
 caffe_option(MKL_MULTI_THREADED  "Use multi-threading"   ON IF NOT MKL_USE_SINGLE_DYNAMIC_LIBRARY)
 
 # ---[ Root folders
-set(INTEL_ROOT "/opt/intel" CACHE PATH "Folder contains intel libs")
+if(WIN32)
+  set(ProgramFilesx86 "ProgramFiles(x86)")
+  set(INTEL_ROOT $ENV{${ProgramFilesx86}}/IntelSWTools/compilers_and_libraries/windows CACHE PATH "Folder contains intel libs")
```

Comment:
If you are taking about https://github.com/BVLC/caffe/blob/master/cmake/Dependencies.cmake#L17-L30 

The default windows build links to openblas which does not need OpenMP to link properly. Maybe MKL needs it I have never tested caffe with MKL.